tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -0,0 +1,115 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This kubevirt test relies on the upstream test code, which is downstreamed as virt-tests.
+#          This is the part running on agent node.
+# Maintainer: Nan Zhang <nan.zhang@suse.com>
+
+use base multi_machine_job_base;
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use utils;
+
+sub run {
+    my ($self) = shift;
+    my $sut_ip = get_required_var('SUT_IP');
+
+    set_var('AGENT_IP', $sut_ip);
+    bmwqemu::save_vars();
+
+    # Synchronize the server & agent nodes
+    barrier_create('KUBEVIRT_TEST_SETUP', 2);
+    barrier_wait('KUBEVIRT_TEST_SETUP');
+    # mutex_create('KUBEVIRT_TEST_SETUP');
+
+    my $server_ip = $self->get_var_from_child("SERVER_IP");
+    record_info('Server IP', $server_ip);
+
+    record_info('Start RKE2 agent setup');
```

Comment:
Please pay attention to record_info usage
```
record_info($title, $output [, result => $result] [, resultname => $resultname]);
```
Both title and output are 'must have', otherwise you will have warnings in your log.

Reply:
OK, updated.